src/sign-verify/clu_crl_verify.c

```diff
@@ -255,7 +255,7 @@ int wolfCLU_CRLVerify(int argc, char** argv)
                 WOLFCLU_LOG(WOLFCLU_E0, "Verification failed");
                 WOLFCLU_LOG(WOLFCLU_E0, "Err (%d) : %s",
                     err, wolfSSL_ERR_reason_error_string(err));
-                ret = WOLFCLU_FATAL_ERROR;
+               // ret = WOLFCLU_FATAL_ERROR;
```

Comment:
these might be commented out just for debugging?

Reply:
Nice catch. Error were removed so that crl files are still generated when verification/input parsing fails

Reply:
This looks to differ from the expected results:

```
$ openssl crl -noout -CAfile client.pem -in ./certs/crl.pem 
Error getting CRL issuer certificate
$ echo $?
1
$ ./wolfssl crl -noout -CAfile client.pem -in ./certs/crl.pem
Verification failed
Err (-190) : ASN CRL no signer error to confirm failure
$ echo $?                                                    
0
```

Otherwise the PR seems good. Will follow up on this with another PR.